engine-tests/src/utils/solidity/uniswap.rs

```diff
@@ -214,17 +214,17 @@ impl PositionManager {
             .function("mint")
             .unwrap()
             .encode_input(&[ethabi::Token::Tuple(vec![
-                ethabi::Token::Address(params.token0.raw()),
-                ethabi::Token::Address(params.token1.raw()),
+                ethabi::Token::Address(params.token0.raw().0.into()),
```

Comment:
Looks like there is a lot of this kind of type conversion now. I assume this is because `ethabi` is no longer actively maintained and thus depends on an old version of the Ethereum primitive types. I wonder if we should look into migrating to the `alloy` library some time in the future.

Reply:
Yes. Exactly, that's because the `ethabi` is no longer supported. But it's a good point about the `alloy` crate. We've already discussed it with @mrLSD.